src/algorithms/primeFinder.js

```diff
@@ -0,0 +1,19 @@
+export default function isPrime(number) {
+    var start = Math.ceil(number / 2);
+    for (let i = start; i >= 2; i--) {
+        if (!(number%i)) {
+            return false;
+        }
+    }
+    return true;
+}
+export function primeFactors(number) {
+    var start = Math.ceil(number / 2);
+    var factors = {};
```

Comment:
what benefit does giving your answer as an object give? 
Why not use an array?
Generally use objects for data that comes in the form of a key value pair. Arrays are for lists of items. If you ever find yourself using an index as a key, give some good hard thought to why you're using an object. maybe it's still the right choice, but you better have a damn good reason.